satpy/multiscene.py

```diff
@@ -229,7 +229,7 @@ def is_generator(self):
     @property
     def loaded_dataset_ids(self):
         """Union of all Dataset IDs loaded by all children."""
-        return set(ds_id for scene in self.scenes for ds_id in scene.keys())
+        return {ds_id for scene in self.scenes for ds_id in scene.keys()}
```

Comment:
I'd rather have this explicitly as a `set` as curly braces can easily be mistaken for dicts.